src/core/scaler.js

```diff
@@ -44,5 +44,5 @@ export function init(stageHeight, game) {
     );
 
     resize();
-    window.onresize = fp.debounce(200, resize);
+    window.onresize = resize;
```

Comment:
this rings a bell, are you sure it doesn't break anything?

Reply:
will be tested to see if it breaks anything, don't know what it used to break, but using it breaks something else too!

Reply:
I've looked through the history but all the commits are vague :p